### PR TITLE
NEW: alpha diversity tutorial with LME

### PR DESCRIPTION
**Pull request by ebolyen**

### CODE REVIEW

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (ebolyen) [inline book/030-tutorial-downstream/060-alpha-diversity.md:18]: Oh, this is stale.

**Comment by ebolyen** [comment]
Doc build failure is a result of a stale env, that should self-correct soon.

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:20]: Missing a period (or colon) at the end of this sentence.

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:35]: I don't think you need *almost* here, right? We are violating the assumptions of the test.

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:38]: _independence_ -> _samples to be independent_

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:41]: spelling: reassuringly

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:45]: if -> whether

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:53]: interpret -> answer _or_ address _or_ explore ...

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:32]: I suggest replacing "alpha diversity" and "diversity" with "richness" throughout this document.

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:64]: diversity -> richness

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:77]: It's -> Its

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:80]: I think:
are -> it is

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:110]: to the -> the

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:132]: It's actually most likely the extreme antibiotic treatment they receive pre- and peri-HCT. So you could say "primarily related to the bone marrow transplant procedure" (so it's referring to the whole treatment, rather than just the transplant itself), or get more specific if you want to.

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:174]: In some of the earlier chapters I posed challenges to create a command, and then hide the answer in the dropdown admonition (see the admonition just above [this heading](https://qiime2-workshops.s3.us-west-2.amazonaws.com/faes-jan2022/030-tutorial-downstream/010-filtering.html#filter-features-from-sequence-data-to-reduce-runtime-of-feature-annotation)). This would fit really well here.

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (gregcaporaso) [inline book/030-tutorial-downstream/060-alpha-diversity.md:130]: transplant -> transplant protocol on gut health

**Review by gregcaporaso — CHANGES_REQUESTED** [review]
@ebolyen, this is looking really good, nice work! 

I haven't seen the outputs from this yet - I think we're still waiting on an environment file, or at least a build with the new environment file? I'm curious how the last LME looks compared to the same model except using the "days since FMT" as the state column. That should show a stronger effect I think, since the richness increase will be better aligned among the treatment recipients (it was not a fixed time from the HCT, but rather performed when it was thought the patients would best tolerate it). If you haven't compared those yet, I think we should do that before we get to the final version of this document (we can open an issue for that, doesn't need to block merge of this PR). 

I don't need to review this one again before merge, but I'm happy to if you want additional input.

**Comment by thermokarst** [comment]
New env files are up, I just triggered a rebuild.

**Comment by ebolyen** [comment]
Thanks for the review @gregcaporaso!

Re: protocol vs transplant, that makes a lot of sense. I had definitely misunderstood where the effect was. That was also the reason I went with HCT instead of FMT in the final plot (which looked fine, but FMT is actually even better, as you expect).

I can make these changes.

Here's the final model FMT generates (filename is wrong, since it was from an earlier revision before I went with HCT):
https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fdl.dropbox.com%2Fs%2Fid4xm95pf8mrm0o%2Flme-obs-features-interactions%25281%2529.qzv%3Fdl%3D1

File: book/030-tutorial-downstream/060-alpha-diversity.md
Comment (ebolyen) [reply book/030-tutorial-downstream/060-alpha-diversity.md:80]: Fixed by making the sentence plural.

**Comment by ebolyen** [comment]
Latest updates require this to have propagated into an env: https://github.com/qiime2/q2cli/pull/275/files
So tune in tomorrow I guess.

**Pull request merged**
